### PR TITLE
Allow field and paragraph classifications without labels

### PR DESCRIPTION
**Pull request by jotare**
### Description
Describe the proposed changes made in this PR.

### How was this PR tested?
Describe how you tested this PR.

### CODE REVIEW

File: nucliadb/nucliadb/train/generators/field_classifier.py [hunk 1]
```diff
@@ -65,19 +65,21 @@ async def generate_field_classification_payloads(
...
 ) -> AsyncIterator[FieldClassificationBatch]:
-    labelset = f"/l/{trainset.filter.labels[0]}"
-
     # Query how many resources has each label
     request = StreamRequest()
     request.shard_id.id = shard_replica_id
-    request.filter.tags.append(labelset)
-    total = 0
 
+    labelset = None
```
Comment (tarekziade) [inline nucliadb/nucliadb/train/generators/field_classifier.py:72]: At this point should we only have 0 or 1 item in `trainset.filter.labels` ?

Comment (jotare) [reply nucliadb/nucliadb/train/generators/field_classifier.py:72]: Yes

**Review by tarekziade — COMMENTED** [review]
looks like we're changing `generate_field_classification_payloads` behavior but I don't see any tests, would it be possible to exercise it?

**Review by tarekziade — APPROVED** [review]
Thanks for the test, you rock

**Pull request merged**